_inc/client/at-a-glance/scan.jsx

```diff
@@ -181,6 +168,21 @@ class DashScan extends Component {
 		} );
 	}
 
+	getUpgradeBanner() {
+		return (
+			<JetpackBanner
+				callToAction={ __( 'Upgrade' ) }
+				title={ __( 'Find threats early so we can help fix them fast.' ) }
+				disableHref="false"
```

Comment:
Does this `disableHref` prop do anything? Seems like we only pass it here and there but it has no real purpose.